src/js/controller/model.js

```diff
@@ -587,6 +586,9 @@ const Model = function() {
             playPromise = loadAndPlay(this, item);
             playAttempt(this, playPromise, playReason);
         } else {
+            if (item.starttime) {
+                _provider.seek(item.starttime);
+            }
             playPromise = _provider.play() || resolved;
```

Comment:
Providers should handle seeking to item.starttime. Doing this in the model will have side effects on different providers. 

If, and only if, the video’s arc is changed externally, then the detached mediamodel’s setup state should be set back to false.

Reply:
*src (autocorrected to arc)